src/bayesdb.py

```diff
@@ -142,7 +144,7 @@ def execute(self, string, bindings=None):
         if bindings is None:
             bindings = ()
         if self.tracer:
-            self.tracer(string, bindings)
+            finish_thunk = self.tracer(string, bindings)
```

Comment:
This was intended on being compatible with tracers that don't have a post-tracer functionality, which would be required to return None.